src/handlers/teams.ts

```diff
@@ -14,19 +14,50 @@
  * limitations under the License.
  */
 
+// Docs and markdown: https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/connectors-using?tabs=cURL
+
+
 import { AdaptiveCards } from "@microsoft/adaptivecards-tools";
 import { WebhookTarget } from "../WebHookTarget";
 import { IqWebhookPayloadApplicationEvaluation, IqWebhookPayloadWaiverRequest } from "../types";
 import { BaseHandler } from "./base";
-
 import template from "../templates/adaptive-card-default.json";
 import { getIqUrlForApplicationEvaluation } from "../helpers/iq";
 import { HandlerNotImplementedError } from "../error"
 
 export class TeamsHandler extends BaseHandler {
     
     public handleWaiverRequest(payload: IqWebhookPayloadWaiverRequest, target: WebhookTarget): void {
-        throw new HandlerNotImplementedError("Method not implemented.")
+        const color = "ff8400" // Orange
```

Comment:
constants that may be reused should be declared in a more shared place - e.g.  `constants.ts` ?

Reply:
Done- and then some! `constants.ts line 27`